### DISCUSSION TITLE
Misleading wording “added a commit” in an issue that I mentioned in a comment on a commit

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#19244] [Mobile]
I mentioned the issue <https://github.com/jenkins-infra/helpdesk/issues/3015> in the comment <https://github.com/jenkins-infra/update-center2/commit/d02e5ec15d67d98aca3ad8d42292b7648cbee357#r76725174> on a commit made by another person, and now when I look at the issue using the GitHub mobile app, it claims I "added a commit". But I did not author, commit, push, or merge the commit; I only commented on it.

![image](https://user-images.githubusercontent.com/46201428/175218226-13e6e09e-f160-4c4c-9cbc-17382b2080d6.png)

The web UI uses the more correct wording "referenced this issue":

![image](https://user-images.githubusercontent.com/46201428/175220205-a697c3e6-efe9-4c47-8ad0-1bd462f0a99e.png)

**Comment by Iron-Ham** [comment]
Thank you for reporting this! I'll create an internal issue to track this.
